src/index.ts

```diff
@@ -158,32 +159,37 @@ class CruxPayPeer extends EventEmitter {
         // Setting up the default modules as fallbacks
         this._storage =  this._options.storage || new storage.LocalStorage();
         this._encryption = this._options.encryption || encryption.Encryption;
-        this._nameservice = this._options.nameservice;
+        this._nameService = this._options.nameService;
         this.walletClientName = this._options.walletClientName;
 
         log.info(`Config mode:`, config.CONFIG_MODE);
         log.info(`CruxPayPeer Initialised`);
     }
 
     public async init() {
-        const configService = new BlockstackConfigurationService(this.walletClientName);
-        await configService.init();
-        if (!this._nameservice) {
-            this._nameservice = await configService.getBlockstackServiceForConfig();
-        }
+        let configService;
 
         if (this._hasPayIDClaimStored()) {
             const payIDClaim = this._storage.getJSON("payIDClaim");
             // log.debug(`Local payIDClaim:`, payIDClaim)
             this._setPayIDClaim(new PayIDClaim(payIDClaim as ICruxPayClaim, { getEncryptionKey: this._getEncryptionKey }));
+            const ns: blockstackService.BlockstackService = new blockstackService.BlockstackService();
+            const status = await ns.getRegistrationStatus({secrets: (this._payIDClaim as PayIDClaim).identitySecrets});
```

Comment:
status will always return NONE unless identity is restored with the method `restoreIdentity` which will set the private property `_identityCouple`.